src/sidebar/components/test/markdown-editor-test.js

```diff
@@ -96,9 +100,7 @@ describe('MarkdownEditor', () => {
     describe(`"${command}" toolbar command`, () => {
       it('applies formatting when toolbar button is clicked', () => {
         const onEditText = sinon.stub();
-        const wrapper = mount(
-          <MarkdownEditor text="test" onEditText={onEditText} />
-        );
+        const wrapper = createComponent({ text: 'test', onEditText });
```

Comment:
don't need `text: 'test'`

Reply:
Ah, I had actually left this intentionally to make it easier to see that the test inputs and assertions corresponded. I have pushed a change to make this more obvious by using a variable rather than repeating the `'test'` string.